zio-http/src/main/scala/zhttp/http/Http.scala

```diff
@@ -256,6 +256,51 @@ sealed trait Http[-R, +E, -A, +B] extends (A => ZIO[R, Option[E], B]) { self =>
   final def provideSome[R1](r: R1 => R)(implicit ev: NeedsEnv[R]): Http[R1, E, A, B] =
     Http.fromPartialFunction[A](a => self(a).provideSome(r))
 
+  /**
+   * Provides layer to HttpApp.
+   * {{{
+   * private val app: HttpApp[Random, Nothing] = ???
+   *
+   * val layer: ZLayer[Any, Nothing, Random] = ???
+   * val newApp: Http[Any, Nothing, Request, Response[Random, Nothing]] =
+   * app.provideLayer(layer)
+   * }}}
```

Comment:
Scala doc seemed to be messed up because of scalafmt. Keep it simple, just a description.